libs/utils/utils.js

```diff
@@ -1231,3 +1231,26 @@ export function loadLana(options = {}) {
 }
 
 export const reloadPage = () => window.location.reload();
+
+export function getVideoIntersectionObserver() {
```

Comment:
This is probably not a good fit for the utils file, it should be defined closer to where it's needed.

Reply:
@overmyheadandbody @narcis-radu 
Moved `getVideoIntersectionObserver` function to `decorate.js`